src/common/components/giftViews/giftListItem/giftListItem.tpl.html

```diff
@@ -0,0 +1,28 @@
+<div class="col-xs-12 col-sm-3 mb">
+  <img desig-src="{{$ctrl.gift['designation-number']}}">
+</div>
+
+<div ng-class="{'col-xs-12 col-sm-9': !$ctrl.selectable, 'col-xs-7 col-sm-6': $ctrl.selectable}">
+  <h4 class="recipient-name">{{$ctrl.gift['designation-name']}}</h4>
+  <span class="modal-meta">#{{$ctrl.gift['designation-number']}}</span>
+  <ng-transclude />
+</div>
+
+<div ng-if="$ctrl.selectable"
+     class="col-xs-5 col-sm-3">
+  <div class="checkbox-right" ng-transclude="selectInput" ng-switch="$ctrl.selectable">
+    <label ng-switch-when="checkbox">{{$ctrl.selectLabel}}
+      <input type="checkbox"
+             name="selectedGift"
+             ng-model="$ctrl.gift.$$selectedGift"
+             ng-change="$ctrl.onSelected({gift: $ctrl.gift, selected: $ctrl.gift.$$selectedGift})">
```

Comment:
What are these for? I guess I'm not immediately seeing how data is getting out of this component. Is it by updating the gift object or by calling onSelected? It doesn't look like the onSelected binding is hooked up here https://github.com/CruGlobal/give-web/pull/190/files#diff-119da8a34d16697efc4a9919ad4bc57eR24.